source/reference/system-requirements.rst

```diff
@@ -39,7 +38,7 @@ these features:
 * Instruction Set:
 
   - 64-bit
-  
+
 * Instruction Set Extensions:
```

Comment:
Should "Intel® Streaming SIMD Extensions 4.1 (Intel® SSE 4.1)" stand after above revision? Ln 45

Reply:
The order here is fine. 4.2 implies 4.1 which implies 3.0, so I would leave it.